core/mixins/_media-queries.scss

```diff
@@ -97,3 +105,16 @@
     @content;
   }
 }
+
+
+/**
+ * Retina.
+ */
+
+@mixin respond-retina($resolution: $retina-resolution) {
+  $resolution: if(unitless($resolution), $resolution * 96dpi, $resolution);
+  
+  @media (min-resolution: $resolution) {
+    @content;
+  }
+}
```

Comment:
> Also should we mention the mixins will accept the units `dppx` and `dpcm`?

It turns out that Autoprefixer doesn't support `dpcm` at all - it doesn't generate any fallbacks, and when using `dppx`, it doesn't generate the `dpi` fallback. So, while you could quite happily enter these units, it's not recommened yet.